apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js

```diff
@@ -37,85 +37,81 @@ const FindOrganizationsByAddressService = new GQLCustomSchema('FindOrganizations
         },
         {
             access: true,
-            type: 'type FindOrganizationByAddressOrganizationType { id: ID!, name: String!, tin: String!, type: OrganizationTypeType! }',
+            type: 'type FindOrganizationByAddressReceiptType { accountNumber: String, category: ID!, balance: String, routingNumber: String, bankAccount: String, address: String }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationByAddressMeterValuesType { value1: String!, value2: String, value3: String, value4: String }',
```

Comment:
Keep the values flat as `Meter` schema does.

Meter.value1, Meter.value2, not Meter.values.value1